docs/source/raft_configuration.md

```diff
@@ -330,5 +330,26 @@ channels (this does not mean they share data --- they do not!), if a Raft node i
 part of the consenter set in many channels, you might want to lengthen the amount
 of time it takes to trigger an election to avoid inadvertent leader elections.
 
+2. As aforementioned, write-ahead logs (wal) and snapshots should be persisted in
+order to attain safety proerty of Raft protocol. However, it may happen if a node
+loses its data by accident, and the node would panic with following message:
+```
+tocommit(X) is out of range [lastIndex(Y)]. Was the raft log corrupted, truncated, or lost?
+```
+Human intervention is required to resurrect problematic node in this case:
+- Option 1: remove node and readd it to consenter set via config update, following
+instructions in reconfiguration section in this doc. This is the most appropriate
+option and does not violate safety property of Raft protocol.
+- Option 2: wipe out wal and snapshot directories of problematic channel, restart
+current leader node, then restart corrupted orderer. This essentially treats corrupted
```

Comment:
i think we first need to restart the corrupted orderer, no? What if the leader is restarted and then the node says its index is high because it still hasn't been restarted?

Reply:
When this problem occurs, the corrupted node should already be down due to panic, therefore not very important to emphasize on the order.